superset/config.py

```diff
@@ -1452,7 +1452,7 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
 
 # URI to database storing the example data, points to
 # SQLALCHEMY_DATABASE_URI by default if set to `None`
-SQLALCHEMY_EXAMPLES_URI = None
+SQLALCHEMY_EXAMPLES_URI = "sqlite:///" + os.path.join(DATA_DIR, "examples.db")
```

Comment:
@dpgaspar are we supporting sqlite as a valid examples db in production?

Reply:
same as before since our default for the metadata db is also sqlite https://github.com/apache/superset/blob/master/superset/config.py#L187
The difference is before this PR both schemas were on the same sqlite db

Reply:
We have an approved SIP to [drop support for SQL Lite](https://github.com/apache/superset/issues/8874). I've added a [ticket](https://github.com/orgs/apache/projects/292/views/1?pane=issue&itemId=40454778) to the 4.0 board to ensure we execute this in the next major release.

Reply:
We do discourage people to use sqlite, though, so I'm wondering if we should put something in UPDATING.md that encourage people to change this value to a different db type?